<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="EC2AMAZ-S11DBRH" failures="1" tests="3" name="runner.testCase1Runner" time="66.712" errors="0" timestamp="2024-02-14T07:03:25 GMT" skipped="0">
  <testcase classname="runner.testCase1Runner" name="runScenario" time="35.894">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepDefinations.TestCase1stepDefination.item_must_be_listed(TestCase1stepDefination.java:95)
at âœ½.Item must be listed(file:///C:/Users/Administrator/Documents/GitHub/Selanium/CucumberTestCase2/src/test/resources/features/testcase1.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.testCase1Runner" name="runScenario" time="30.467"/>
  <testcase classname="runner.testCase1Runner" name="runScenario" time="0.351"/>
</testsuite> <!-- runner.testCase1Runner -->
